src/Label/Events/CopyCreated.php

```diff
@@ -38,7 +39,8 @@ public static function deserialize(array $data): CopyCreated
             $data[self::NAME],
             new Visibility($data[self::VISIBILITY]),
             new Privacy($data[self::PRIVACY]),
-            new UUID($data[self::PARENT_UUID])
+            new UUID($data[self::PARENT_UUID]),
+            (bool) $data[self::EXCLUDED]
```

Comment:
I think this will result in a crash when trying to modify old labels which were not created with an excluded field. This is a breaking change.